### PR TITLE
Feature tnw

### PR DESCRIPTION
**Pull request by Ramon-Carrillo**
Please check our work.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,788 @@
...
+					<div> <a class="o_color" href="#"> Get updates by email</a><span>|</span></div>
+					<div><a class="o_color" href="#"> See more</a></div>
+				</div>
+			</div>
+			<div class="d_grid grid j_content h_100">
+				<div class="grid_item d_flex f_dir h_100">
+					<div class="funding_1 d_grid j_content">
+						<div class="funding_description h_100"></div>
+						<!--<div class="fundingimg d_flex">-->
+						<img class="w_100"
```
Comment (kevincappuccino) [inline index.html:221]: - [ ] It’s better to delete **commented piece of codes**. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))  
:x: **commented piece of codes** /* `.nav { color: red;}` */  
:heavy_check_mark: **comments about code** /*Navigation */

File: index.html [hunk 2]
```diff
@@ -0,0 +1,788 @@
...
+						<div class="price d_flex f_dir w_100 h_100">
+							<a class="box1" href="#">
+								<h5 class="dark w_100">$5M</h5>
+							</a>
+							<a class="box1" href="#">
+								<h5 class="light w_100">SERIES A</h5>
+							</a>
+						</div>
+					</div>
+					<!--</div>-->
```
Comment (kevincappuccino) [inline index.html:234]: - [ ] It’s better to delete **commented piece of codes**. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))  
:x: **commented piece of codes** /* `.nav { color: red;}` */  
:heavy_check_mark: **comments about code** /*Navigation */

File: index.html [hunk 3]
```diff
@@ -0,0 +1,788 @@
...
+								AlbionVC, as COVID-19 accelerates
+								remote clinical studies <span> on TechCrunch</span></a></p>
+					</div>
+				</div>
+
+				<div class="grid_item d_flex f_dir h_100">
+					<div class="funding_1 d_grid j_content">
+						<div class="funding_description h_100"></div>
+						<!--<div class="fundingimg d_flex">-->
+						<img class="w_100"
```
Comment (kevincappuccino) [inline index.html:257]: - [ ] It’s better to delete **commented piece of codes**. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))  
:x: **commented piece of codes** /* `.nav { color: red;}` */  
:heavy_check_mark: **comments about code** /*Navigation */

File: index.html [hunk 4]
```diff
@@ -0,0 +1,788 @@
...
+						<div class="price d_flex f_dir w_100 h_100">
+							<a class="box1" href="#">
+								<h5 class="dark w_100">$5M</h5>
+							</a>
+							<a class="box1" href="#">
+								<h5 class="light w_100">SERIES A</h5>
+							</a>
+						</div>
+					</div>
+					<!--</div>-->
```
Comment (kevincappuccino) [inline index.html:270]: - [ ] It’s better to delete **commented piece of codes**. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))  
:x: **commented piece of codes** /* `.nav { color: red;}` */  
:heavy_check_mark: **comments about code** /*Navigation */

File: index.html [hunk 5]
```diff
@@ -0,0 +1,788 @@
...
+								AlbionVC, as COVID-19 accelerates
+								remote clinical studies <span> on TechCrunch</span></a></p>
+					</div>
+				</div>
+
+				<div class="grid_item d_flex f_dir h_100">
+					<div class="funding_1 d_grid j_content">
+						<div class="funding_description h_100"></div>
+						<!--<div class="fundingimg d_flex">-->
+						<img class="w_100"
```
Comment (kevincappuccino) [inline index.html:293]: - [ ] It’s better to delete **commented piece of codes**. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))  
:x: **commented piece of codes** /* `.nav { color: red;}` */  
:heavy_check_mark: **comments about code** /*Navigation */

File: index.html [hunk 6]
```diff
@@ -0,0 +1,788 @@
...
+						<div class="price d_flex f_dir w_100 h_100">
+							<a class="box1" href="#">
+								<h5 class="dark w_100">$5M</h5>
+							</a>
+							<a class="box1" href="#">
+								<h5 class="light w_100">SERIES A</h5>
+							</a>
+						</div>
+					</div>
+					<!--</div>-->
```
Comment (kevincappuccino) [inline index.html:306]: - [ ] It’s better to delete **commented piece of codes**. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))  
:x: **commented piece of codes** /* `.nav { color: red;}` */  
:heavy_check_mark: **comments about code** /*Navigation */

File: index.html [hunk 7]
```diff
@@ -0,0 +1,788 @@
...
+						<p><a href="#">MedTech startup uMotif raises £5m from
+								AlbionVC, as COVID-19 accelerates
+								remote clinical studies <span> on TechCrunch</span></a></p>
+					</div>
+				</div>
+				<div class="grid_item d_flex grid_4 f_dir h_100">
+					<div class="funding_1 d_grid j_content">
+						<div class="funding_description h_100"></div>
+						<!--<div class="fundingimg d_flex">-->
+						<img class="w_100"
```
Comment (kevincappuccino) [inline index.html:328]: - [ ] It’s better to delete **commented piece of codes**. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))  
:x: **commented piece of codes** /* `.nav { color: red;}` */  
:heavy_check_mark: **comments about code** /*Navigation */

File: index.html [hunk 8]
```diff
@@ -0,0 +1,788 @@
...
+							<a class="box1" href="#">
+								<h5 class="dark w_100">$5M</h5>
+							</a>
+							<a class="box1" href="#">
+								<h5 class="light w_100">SERIES A</h5>
+							</a>
+						</div>
+					</div>
+					<!--</div>-->
+					<div class="funding_2 d_flex f_dir w_100">
```
Comment (kevincappuccino) [inline index.html:342]: - [ ] It’s better to delete **commented piece of codes**. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))  
:x: **commented piece of codes** /* `.nav { color: red;}` */  
:heavy_check_mark: **comments about code** /*Navigation */

**Review by kevincappuccino — CHANGES_REQUESTED** [review]
# [STATUS] CHANGES REQUIRED♻️

#### Hey team,  your project is looking really good, however, to accomplish as the actual page, you required to do these changes, listed below:
 
_Note_: That changes marked as [OPTIONAL] are not required to approve your project, but I strongly recommend that you guys go for that extra mile, to get more knowledge and have a better project to add at your portfolio.
 
 
## [REQUIRED CHANGES]
#### Clone REQUIREMENTS 👥 (Those adjusts to make your page really look like the real one)⤵
#### Large screens
Img1 ⤵
- [x] Decrease those margins
- [x] Fix that image width (to make it more similar to the original one)
- [x] Apply a blur on the background of those images to make it easier to read the text
- [x] Remove that horizontal bar
image
Img2 ⤵
- [x] fix that iamge size
- [x] decrease that gap betwen those images and that sections below
- [x] use sans-serif fonts
image
Img3⤵
- [x] Add margin-top please.
image

#### Small screens
- [x] Move that section (add info, get updates, see more) to below that Latest founding
- [x] Fix typo Lastes ➡ "Latest"
- [ ] Add margin left and right
- [ ] remove that horizontal part
image

#### Good practices REQUIREMENTS (Those changes to make you code like a pro 👽)⤵
- [x] Add a descriptive and meaningful description to your pull request. For more info check [here](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/)
Tip: Describe what you guys did in this project; Like: We used CSS flexbox, float and grid; we added a sidebar, we used semantic HTML. In real world: PR Should be descriptive, so the reviewer can know what you guys are trying to change at the main project.
image

- [x] Guys, next time check up the way you write some commits, they should be descriptive so the other programmer can see exactly what you changed with that commit. [Check this link](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8/)
For example, look at some of those commits⤵
image
We can't know what did you have changed. Please improve them 💪
- [x] Kindly remember to remove all the unnecessarily repeated CSS code. Keeping your code DRY is an excellent practice that will help you a lot and make your code more professional. (Check this link form better understanding [Good and Bad CSS Practices for Beginners](https://speckyboy.com/good-bad-css-practices/)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)
Take a look at this example:
image
Your HTML will be something like this: 
image
Apply the same logic above in **ALL** those repeated rules listed below:
image

- [x] It’s better to delete **commented piece of codes**. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))  
:x: **commented piece of codes** /* `.nav { color: red;}` */  
:heavy_check_mark: **comments about code** /*Navigation */  


[OPTIONAL REQUIREMENTS]
- [ ] Add ribbons and prices
image
## Please make the required changes and resubmit for a new code review once you are ready💪

## As always, I wish you all the very best and see you both at the top🚀


👨🏻‍💻 **TSE: Kevin Alves**

  

📫 Have any question? Better call Kevin:

<a  href="mailto:kevinalves.js@gmail.com"> gmail</a> [Twitter Follow](https://twitter.com/intent/follow?screen_name=kevinnnmn) [](https://microverse-students.slack.com/team/UT86D3AP4)

<sub>*We're here to help you, to ensure that you're using the best practices and getting you ready for the job market. In real-world, you'll have a lot of changes requests for your real clients, it's good to get used to it.</sub>

**Comment by Kingobaino1** [comment]
- As requested, the changes have been effected. However, it was observed that after creating classes for some properties and assigning them to the HTML tags, it will not render in the page. (e.g. we created a class called cover to host the property background-size: cover, but if you assign this class to the HTML tag, it won't render in the page.)

**Review by Spayco — CHANGES_REQUESTED** [review]
# Changes Requested 

Hello! 👋 
Awesome job so far!! 👍 you still need to make some changes though! 

## Required Changes ♻️ 

* [ ] Pull Request has a meaningful title and a brief description ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_prs.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))

- [ ] The Navbar should have the links on the left side, and the search bar on the right, yours are not well placed

| Yours | Original |
| --- | --- |
| image | image |

on small screens

- [ ] The yellow bar breaks

image



### please make the appropriate changes and ask for another code review ✔️ 

Happy Coding!

Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

**Comment by Kingobaino1** [comment]
**Completion of required change**
- adjusted navbar as requested
- repaired breakpoint for iPhone x

**Review by elmejdki — APPROVED** [review]
# status: approved :heavy_check_mark: 
## Great work guys, you have made a really good job

now you can merge to master your project is approved

## Happy Coding :tophat:

**Pull request merged**